lib/messages.js

```
@@ -31,8 +32,9 @@ exports.parseMessage = function(buffer, callback){
             callback(null, 0);
         }
         else {
-            var messageType = buffer.readUInt8(headerSize);
-            var startOffset = headerSize + 1; // 1 byte message type
+            var version = buffer.readUInt8(headerSize); // 1 byte version
```

Let's throw an error if the version does not match